data-collections-exercises.md

```diff
@@ -61,37 +61,41 @@ Select all elements in `heroes` that begin with "B" using a method. [The Ruby do
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+ninja_turtle = { 
+                 name: "Michelangelo",
+                weapon: "Nunchuks", 
+                radical: true,
+                }  
 ```
 
 Add a key `age` to `ninja_turtle`. Set it to whatever numerical value you'd like.
 
 ```rb
-# Your answer here
+ninja_turtle.merge(age: "1000")
```

Comment:
when you use merge you need to use the `!` if you want to modify the hash directly or else it will make a copy and add the changes to that copy